### PR TITLE
refactor: split translate cmd from logto cli

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Split translate command from `@logto/cli`.  Will be maintained as a standalone `@logto/translate` package

## Testing

Tested locally

## Checklist

- [x] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

File: packages/translate/package.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  },
+  "dependencies": {
+    "@logto/core-kit": "workspace:^2.5.0",
+    "@logto/language-kit": "workspace:^1.1.0",
+    "@logto/phrases": "workspace:^1.13.0",
+    "@logto/phrases-experience": "workspace:^1.7.0",
+    "@logto/shared": "workspace:^3.1.1",
+    "@silverhand/essentials": "^2.9.1",
+    "chalk": "^5.3.0",
+    "dotenv": "^16.4.5",
```
Comment (gao-sun) [inline packages/translate/package.json:49]: installed but not in use?

File: package.json [hunk 2]
```diff
@@ -52,6 +52,7 @@
     }
   },
   "dependencies": {
-    "@logto/cli": "workspace:^1.1.0"
+    "@logto/cli": "workspace:^1.1.0",
+    "@logto/translate": "workspace:^0.0.0"
```
[the remark this answers is not in the document]
Comment (charIeszhao) [reply package.json:56]: You don't need "script" to invoke the cli, the pnpm monorepo finds it by its package name. So the following command still works:
```
pnpm translate <command> [options]
```

File: packages/translate/package.json (hunk 1 above)
Comment (charIeszhao) [reply packages/translate/package.json:49]: You're right. I missed this one

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (charIeszhao) [reply package.json:57]: Wait, I may be wrong. Just tried today on my home computer and it doesn't work anymore. But couple days ago it DID work on my laptop... Weird

Anyways, added a "translate" script.

File: .changeset/silly-hotels-greet.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@logto/translate": minor
```
Comment (gao-sun) [inline .changeset/silly-hotels-greet.md:2]: can you mark `@logto/cli` as well? also we should point out why we don't use major for this breaking change

**Review by gao-sun — APPROVED** [review]

File: .changeset/silly-hotels-greet.md (hunk 3 above)
Comment (charIeszhao) [reply .changeset/silly-hotels-greet.md:3]: Yeah, updated

**Review by wangsijie — APPROVED** [review]

**Pull request merged**